gettsim/data/e_st_abzuege.yaml

```diff
@@ -553,21 +585,21 @@ hhfreib:
       note:
     2015-01-01:
       value: 1908
-      note: geändert durch BGBl. Nr. 30, 16.07.2015
+      note: Art. 1 G. v. 16.07.2015 BGBl. I S.1202
 hhfreib_ch:
   name:
-    de: Entlastungsbetrag für Alleinerziehende, Zusatzbetrag pro Kind ab dem 2. Kind  §24b (2) S.2 EStG
-    en:
+    de: Entlastungsbetrag für Alleinerziehende, Zusatzbetrag pro Kind ab dem 2. Kind
```

Comment:
This is a bit too long for a name. If you touch this in this PR: Can you move parts of that to the description? You can also simply keep the old value/string and do the cleanup in a separate PR.